src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/Set-AzConnectedKubernetes.ps1

```diff
@@ -395,6 +411,7 @@ function Set-AzConnectedKubernetes {
         }
 
         # Parse value from inputObject
+        # ArcAgentryConfiguration is handled in a separate block
         if ($null -ne $InputObject) {
```

Comment:
We came cross the exact same example when we generating Set- internal function.  In the Relay example when generating internal Set-, it replaces the name of parameter "Parameter" (yes, that's the name) of type IWcfRelay to "Inputobject", possibly because the definition is different. e.g. they've used [`x-ms-client-flatten`](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-client-flatten) that no longer recommended. And we adopted the ugly inputObject here ;/